extensions/campeonato-brasileiro/src/topScorers.tsx

```diff
@@ -0,0 +1,98 @@
+import { Action, ActionPanel, Icon, Image, List } from "@raycast/api";
+import moment from "moment";
+import { flag } from "country-emoji";
+import { useState } from "react";
+import useTopScorers from "./hooks/useTopScorers";
+
+function getAge(date: string) {
+  const birthDate = moment(date, "YYYY-MM-DD");
+  const currentDate = moment();
+  const age = currentDate.diff(birthDate, "years");
+  return age.toString();
+}
+function getFormatedDate(date: string) {
+  const parsedDate = moment(date, "YYYY-MM-DD");
+  const formattedDate = parsedDate.format("D MMMM YYYY");
+  return formattedDate.toString();
+}
+
+function getCountryDescription(countryName: string | undefined) {
+  if (countryName == undefined) {
+    return "?";
+  }
+
+  return countryName + " " + flag(countryName);
+}
+
+export default function GetTopScorers() {
```

Comment:
```suggestion
export default function Command() {
```
This is also about what I mentioned about Commands.